packages/core/components/Modal/EditMetadata/PasswordForm.module.css

```diff
@@ -0,0 +1,79 @@
+.error-message {
+    background-color: rgba(194, 48, 48, 0.5);
```

Comment:
there are also the global variables `--error-status-background-color` and `--error-status-text-color` which are what we use for backgrounds/borders in the status messages